nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/storage/PutGCSObject.java

```diff
@@ -194,6 +194,16 @@ public class PutGCSObject extends AbstractGCSProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor GZIPCONTENT = new PropertyDescriptor
+            .Builder().name("gzip.content")
+            .displayName("Allow GZIP Compression")
+            .description("Signals to the GCS Blob Writer whether GZIP compression during transfer is desired. " +
+                    "False means dont gzip and can boost performance in many cases.")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .defaultValue("False")
+            .allowableValues("True", "False")
+            .defaultValue("True")
```

Comment:
The `defaultValue()` is being called twice, with the effective value being set to `True`. This should be corrected to the preferred setting.

Reply:
haha dang it - good catch.  Will resolve.